documentation/meta/release_and_deployment/index.md

```diff
@@ -254,15 +254,20 @@ flowchart TD
 
 #### Force a deployment to a specific version (roll back, redeploy, re-run, etc.)
 
-Roll back and redeployment of the API and frontend follows an identical process,
-based on the ability to force a deployment of the app to a specific version.
-Regardless of whether that version is the version currently running, a past
-version, or a completely new version, the same deployment workflows are used in
-all cases. This process also works to retry a failed deployment.
+```{tip} Quick Tips for Performing a Rollback
+
+To perform a rollback, manually trigger the appropriate deployment workflow from the WordPress/openverse-infrastructure repository using the tag of the latest stable version. You can find the release version number in the [changelogs](/changelogs/index); the tag to pass to the action is the version number prefixed with "rel-", for example "rel-2023.07.03.17.52.00".
```

Comment:
LGTM! Perhaps it's worth adding the link to the infrastructure repo's actions to go quickly?

```suggestion
To perform a rollback, manually trigger the appropriate [deployment workflow](https://github.com/WordPress/openverse-infrastructure/actions)
from the WordPress/openverse-infrastructure repository using the tag of the
latest stable version. You can find the release version number in the
[changelogs](/changelogs/index); the tag to pass to the action is the
version number prefixed with "rel-", for example "rel-2023.07.03.17.52.00".
```

Reply:
Great idea! I think I'll put the link on "WordPress/openverse-infrastructure repository" instead, since we can only link to the actions rather than to a direct workflow 🤔